addons/im_livechat/tests/test_res_users.py

```diff
@@ -12,9 +12,22 @@ def test_livechat_create_res_users(self):
             name="admin_access",
             groups="base.group_erp_manager,base.group_partner_manager",
         )
+        access_user.with_user(access_user.id).create({
+            "login": "test_can_be_created",
+            "name": "test_can_be_created",
+        })
+
+    def test_livechat_create_res_users_livechat_admin(self):
+        access_user = new_test_user(
+            self.env,
+            login="admin_access",
+            name="admin_access",
+            groups="base.group_erp_manager,base.group_partner_manager,im_livechat.im_livechat_group_manager",
+        )
```

Comment:
The change in the test is hiding the issue rather than fixing it.

Members of `group_erp_manager` (even if they are not part of `im_livechat_group_user`) should be able to create/write other users .